server/lib/nl/config_builder/timeline.py

```diff
@@ -25,6 +25,13 @@
 import server.lib.nl.fulfillment.types as types
 
 
+# Get facet id to use
```

Comment:
Maybe add some comments that we use a facet *only* if the date is set (or better yet, rename the function).  Currently that is hidden in the logic...